ui-react/packages/atlasmap-ui/src/AtlasmapUI/MappingTable.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { ReactNode } from 'react';
+import { FunctionComponent } from 'react';
+import { TableIcon, SearchIcon } from '@patternfly/react-icons';
+import { css, StyleSheet } from '@patternfly/react-styles';
+import {
+  Stack,
+  StackItem,
+  Split,
+  SplitItem,
+  Grid,
+  GridItem,
+} from '@patternfly/react-core';
+import { useAtlasmapUI } from './AtlasmapUIProvider';
+import { IAtlasmapField } from './models';
+
+const styles = StyleSheet.create({
+  table: {
+    border: '1px solid #ccc',
+    margin: '10px',
+  },
+  titleBar: {
+    borderBottom: '1px solid #ccc',
+    padding: '10px',
+  },
+  title: {
+    paddingLeft: '10px',
+    fontWeight: 'bold',
+    fontSize: 'larger',
+  },
+  item: {
+    borderBottom: '1px solid #ccc',
+    padding: '10px',
+  },
+  header: {
+    borderBottom: '1px solid #ccc',
+    fontWeight: 'bold',
+    padding: '10px',
+  },
+});
+
+export interface IMappingTableProps {
+  contextToolbar: ReactNode;
+  viewToolbar: ReactNode;
+}
+
+export const MappingTable: FunctionComponent<IMappingTableProps> = ({
+  contextToolbar,
+  viewToolbar,
+}) => {
+  const { mappings } = useAtlasmapUI();
+
+  const mappingRows: Array<ReactNode> = [];
+  mappings.map((mapping, index) => {
```

Comment:
Using `map` and discarding its return value is misleading. A `forEach` would be clearer in its intention here. I'd personally used a `reduce` to avoid having a `const mappingRows` that's then mutated in the `forEach`.